Bitmap tables need glyph ids sorted

Seeing error of the following form:

```
Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/nanoemoji/write_font.py", line 824, in <module>
    app.run(main)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/absl/app.py", line 312, in run
    _run_main(main, args)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/absl/app.py", line 258, in _run_main
    sys.exit(main(argv))
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/nanoemoji/write_font.py", line 818, in main
    _write(ufo, ttfont, font_config.output_file)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/nanoemoji/write_font.py", line 263, in _write
    ttfont.save(output_file)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 181, in save
    writer_reordersTables = self._save(tmp)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 221, in _save
    self._writeTable(tag, writer, done, tableCache)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 634, in _writeTable
    tabledata = self.getTableData(tag)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 656, in getTableData
    return self.tables[tag].compile(self)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/fontTools/ttLib/tables/E_B_L_C_.py", line 181, in compile
    data = indexSubTable.compile(ttFont)
  File "/usr/local/google/home/rsheeter/oss/noto-emoji/venv/lib/python3.9/site-packages/fontTools/ttLib/tables/E_B_L_C_.py", line 456, in compile
    assert all(glyphIds[i] < glyphIds[i+1] for i in range(len(glyphIds)-1))
AssertionError
ninja: build stopped: subcommand failed.
```

Looks to me like sorted color_glyphs on glyph_id in make_cbdt_table would fix this.

--- a/src/nanoemoji/bitmap_tables.py
+++ b/src/nanoemoji/bitmap_tables.py
@@ -173,11 +173,10 @@ def make_cbdt_table(
     color_glyphs: Sequence[ColorGlyph],
 ):
 
-    min_gid, max_gid = reduce(
-        lambda a, c: (min(a[0], c.glyph_id), max(a[1], c.glyph_id)),
-        color_glyphs,
-        (sys.maxsize, -1),
-    )
+    # bitmap tables don't like it when we're out of order
+    color_glyphs = sorted(color_glyphs, key=lambda c: c.glyph_id)
+
+    min_gid, max_gid = color_glyphs[0].glyph_id, color_glyphs[-1].glyph_id
     assert max_gid - min_gid + 1 == len(
         color_glyphs
     ), "Below assumes color gyphs gids are consecutive"

